.gitignore

```
@@ -31,3 +31,9 @@ build/
 
 ### VS Code ###
 .vscode/
+
+# Arquivos de log
+*.log
```

não pegou o arquivo de log pq a regex não tá cobrindo ele. sugiro adicionar as linhas:

```suggestion
*.log
*.log.*
logs/
```